tests/examples/pacer/attachment_pages/gamd_0530995470.html

```diff
@@ -0,0 +1,284 @@
+				var checkbox = elements[i];
+				if(checkbox.name.match(/document_/)){
+					var info = checkbox.value.split('-');//dm_dls_id - file_size - page_size
+					if (checkbox.type == "hidden") {
+						var values = checkbox.name.split('_');
+						var ad_key = values[1] + '_' + values[2];
+						if (ad_checked[ad_key]) {
+							include.push(info[0]);
+						}
+						else {
+							exclude.push(info[0]);
+						}
+					}
+					else {
+						if(checkbox.checked == true){
+							include.push(info[0]);
+						}else{
+							exclude.push(info[0]);
+						}
+					}
+                                }
+                        };
+
+                        if(url){//Link
+                                url += '&exclude_attachments=' + exclude.join(',');
+                                document.location.href = url+'&zipit='+zip;// 0 = Show on screen, 1 = Zip PDF, 2 = Download PDF
+                        }else{
+                                document.getElementsByName('zipit')[0].value = zip;
+                                document.getElementsByName('download_documents')[0].value = include.join(',');
+                                document.view_multi_docs.submit();
+                        }
+                }
+
+                </script>
+	
+			<form><!-- Required for old style javascript, for older browsers. -->
+			<script>CMECF.util.Event.addListener(window, 'pageshow', select_link_documents);</script>
+			<script>CMECF.MainMenu.alternateHelp="////show_multidocs.htm"</script>
+			<table style="table-layout:fixed;word-wrap:break-word;margin: 20px">
+		
+			<tr>
+				<td colspan="2"><b>Document Number:</b></td>
+				<td width="50px"><input type="checkbox" name="document_79_0" onchange="update_sizes(this)" autocomplete="off" value="562021-0-0" /></td>
+				<td><a href="https://ecf.gamd.uscourts.gov/doc1/0531562021" onClick="goDLS('/doc1/0531562021','69687','400','','','1','','','');return(false);">79</a></td>
+				<td></td>
+				<td>0 B</td>
+			</tr>
+			<tr height="50px">
+				<td width="150px" colspan="2"><b>Attachment</b></td>
+				<td width="200px" colspan="2"><b>Description</b></td>
+				<td width="100px"><b>Pages</b></td>
+				<td width="100px"><b>Size</b></td>
+			</tr>
+			
+			<tr>
+				<td width="50px"><input type="checkbox" name="document_79_2" onchange="update_sizes(this)" autocomplete="off" value="631400-58572-5" /></td>
+				<td><a href="https://ecf.gamd.uscourts.gov/doc1/0531631400" onClick="goDLS('/doc1/0531631400','69687','400','','','1','','','');return(false);">2</a></td>
+				<td colspan="2"> recorded deed to secure debt</td>
+				<td>5 pages</td>
+				<td>57.2 KB</td>
+			</tr>
+			  
+			<tr>
+				<td width="50px"><input type="checkbox" name="document_79_2" onchange="update_sizes(this)" autocomplete="off" value="995470-33461-1" /></td>
+				<td><a href="https://ecf.gamd.uscourts.gov/doc1/0531995470" onClick="goDLS('/doc1/0531995470','69687','400','','','1','','','');return(false);">2</a></td>
```

Comment:
Funny one here, this attachment page has 2 attachments both with the exact same index of `2`, we now successfully repair these indexes since the first attachment has a pre-changeover index while the second is a post-changeover index which gives us the correct real indexes of `1` and `2`.